modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
The red bar did not exist at the time of this PR, I think now it has both
I'll get rid of the loader then